src/modules/drop.ts

```diff
@@ -1406,12 +1408,6 @@ class DropV1Module extends ModuleWithRoles<Drop> {
     return await this.sendTransaction("setMaxTotalSupply", [amount]);
   }
 
-  public async setRestrictedTransfer(
```

Comment:
why is this nuked?

Reply:
moved to common interface, method name is still the same.